### DISCUSSION TITLE
Group Assignment #4: Additional interactions

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#72] [Assignments]
Submit:

- Group name
- Group members
- Description of interactivity developed
- Link to maproom

**Comment by alicelu170** [comment]
Group name: Rethink Recycle
Group members: Alice, Hozumi, Emily
Description of interactivity developed: Data table added under Global Map that includes the amount of plastic each country received from the U.S. in 2016, 2017, and 2018. Data can be sorted by year,  country name, and amount. Clicking on an entry will take you to the country on the map.
Link to maproom: https://alicelu170.github.io/DH151/Week%209.project%20work/index.html

**Comment by jomh17** [comment]
Group name: Happy Tweet
Group Members: Ivan, MJ, Josephine
Description: Added a happiness slider and option for the user to input their location. 
Link to site: https://ivanma9.github.io/HappyTweet/index.html

**Comment by ainsleighdouglas** [comment]
Group Name: Team Data
Group members: Ainsleigh, Ryan, Sebastien
Description: Added a graph to compare CO2 output to number of data centers, this data analysis will further enhance our comparison.
Link: https://ainsleighdouglas.github.io/DH-151/Midterm2/visual.html

**Reply by ryanorlando** [reply]
Updated Map: https://ryanorlando.github.io/Dh151/FinalMap/index.html

**Comment by LynnT2** [comment]
Group name: Fantastic Four
Group members: Labiba Alam, Adriana Romero, Lynn Tieu, Haiqi Zhou
Description of interactivity developed: Added a pie chart to visualize the gender distribution from our data. Clicking on a pie slice will filter out the markers on the map.
Link: https://lynnt2.github.io/FantasticFour/Midterm/data.html

**Comment by alexthegreat1** [comment]
- Group name: The cARTographers
- Group members: Kai Colorado & Alex Haddad
- Description of interactivity developed: Added a chart that shows how NAEP scores changed throughout the years for whichever state you hover/click on.
- Link: https://kaicolorado.github.io/DH151-Group/index.html

**Comment by zonaliao** [comment]
Group name: State-of-the-Climate
Group members: Alekhya Vittalam, Natasha Grace, Zona Liao
Description of interactivity developed: Added a bar chart on the right that includes the poverty levels for each country that is clicked on within the map which shows data from 2017, 2016, and 2010. Also added a data table below the map with the location, year, and poverty level for each country. 
Link to maproom: https://alekhyavittalam.github.io/state-of-the-climate/Week9/index.html

**Comment by kmcinerny** [comment]
Group name: Health Mappers
Group members: Kate & Frida
Description of interactivity: Added a slider that maps an outline of the minimum number of mental health facilities in each zip. Also added a chart with racial demographics by zip and buttons that allow users to map either number of programs or those demographics. 
(We plan to include more charts beyond this. We've also started including more info about payment forms for each facility in the point data pop-ups, but we need to continue working on the code.)
Link to maproom: https://kmcinerny.github.io/DH151/Week9%20map

**Comment by danielliuzy** [comment]
Group name: The Musical Maps
Group members: Daniel, Kai and Justis
Description of interactivity developed: Added a chart button that toggles a time based chart, showing the number of shows of the selected genre over the time period of 2019-2020 by month.
Link: https://justisg.github.io/TheMusicalMaps/Website/map.html

**Comment by derekwang99** [comment]
Group Name: Ocean Pollution Posse
Group Members; Eli Aranita, Julia Wood, Derek Wang

Version 1: https://JuliaWood1.github.io/DH151/Week9/index.html
Description of interactivity: By hovering on a country, one would be able to see that country's GDP and the amount of mismanaged plastic waste they produced in 2010.

Version 2: https://eliaranita1.github.io/project1/v3%20GA4/index.html
Description of Interactivity: We were hoping that we could implement a bar chart for each continent. Upon hovering on Asia, for example, the bar chart would show the top five Asian countries that contribute to mismanaged plastic waste.

**Reply by derekwang99** [reply]
Update (Version 2): We got the chart to show by continent, but are having trouble filtering and sorting the data to show the accurate numbers. However, the basic framework is there.

**Comment by michelle14lee** [comment]
Group Name: Datart
Group Members: Annmarie Cong, Eustina Kim, Michelle Lee
Description of Interactivity: We created a bar chart to represent the progression of art over time for each neighborhood.
Link: https://kimjee8955.github.io/Datart/choropleth.html

**Reply by anncong** [reply]
Updated Link: https://kimjee8955.github.io/Datart/index.html

**Comment by lisalou36** [comment]
Group Name: Environmental Racism in Carceral California
Group Members: Hannah Lien, Lisa Kahn, Julia Tanenbaum
Description: We are trying to add an area chart for the geojson layers to show which counties have especially high combinations of toxicity, but we haven't been able to figure out how yet. Also tried adding in a table to display clearly the levels of each toxicity layer for each county which we have made more progress in, but still can't get the data to connect so it's just an empty table right now. Will comment updates if we make any progress.
Link: https://jmtanenbaum.github.io/DHEnviroIncarceration/index.html
